client/hwlib/build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    let os_release_file_path =
+        std::env::var("OS_RELEASE_FILE_PATH").unwrap_or("/etc/os-release".to_string());
+    println!(
+        "cargo:rustc-env=OS_RELEASE_FILE_PATH={}",
+        os_release_file_path
+    );
+}
```

Comment:
are you sure we need this logic since this file should always be `/etc/os_release`?
for tests we can use the same logic as for other paths